libethereum/BlockChain.cpp

```diff
@@ -941,6 +941,7 @@ ImportRoute BlockChain::import(VerifiedBlockRef const& _block, OverlayDB const&
 	{
 		cnote << "SLOW IMPORT:" << _block.info.hash() << " #" << _block.info.number();
 		cnote << "  Import took:" << total.elapsed();
+		cnote << "  gas per second:" << static_cast<double>(_block.info.gasUsed()) / enactment;
```

Comment:
Yes, that's totally fine. I would actually go for integers (or u256) because that is better readable and better parseable for performance analysers.